code/game/jobs/jobs.dm

```diff
@@ -4,16 +4,15 @@ var/const/ENGSEC			=(1<<0)
 var/const/CAPTAIN			=(1<<0)
 var/const/HOS				=(1<<1)
 var/const/WARDEN			=(1<<2)
-var/const/DETECTIVE			=(1<<3)
-var/const/OFFICER			=(1<<4)
-var/const/CHIEF				=(1<<5)
-var/const/ENGINEER			=(1<<6)
-var/const/ATMOSTECH			=(1<<7)
-var/const/AI				=(1<<8)
-var/const/CYBORG			=(1<<9)
-var/const/INTERN_SEC		=(1<<10)
-var/const/INTERN_ENG		=(1<<11)
-var/const/FORENSICS			=(1<<12)
+var/const/OFFICER			=(1<<3)
+var/const/CHIEF				=(1<<4)
+var/const/ENGINEER			=(1<<5)
+var/const/ATMOSTECH			=(1<<6)
+var/const/AI				=(1<<7)
+var/const/CYBORG			=(1<<8)
+var/const/INTERN_SEC		=(1<<9)
+var/const/INTERN_ENG		=(1<<10)
+var/const/FORENSICS			=(1<<11)
```

Comment:
These flags are used by the job preferences.
So changing them wholesale is not a good idea.
I.e. someone who previously had Atmostech on high would now have AI on high.

Just rename DETECTIVE (1<<3) to Investigator and delete FORENSICS (1<<12)

Reply:
The flag for the role is still FORENSICS, so I moved that to the detective slot and deleted the twelfth slot as you said.